src/Tizen.NUI/src/devel/Lite/L.Color.cs

```diff
@@ -46,6 +49,28 @@ public Color(float r, float g, float b, float a)
             A = a;
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Color"/> struct.
+        /// </summary>
+        /// <param name="value">The hex value of 0xRRGGBB format.</param>
```

Comment:
```suggestion
        /// <param name="value">The hex value of 0xRRGGBBAA format.</param>
```

Why RGB? Is RGBA format is more familier to developer?

Reply:
I had same concern about it 👍 

GUI toolkits tend to use alpha value as percentage rather than hex format.
e.g. figma
![image](https://github.com/user-attachments/assets/3520d7c8-3fd2-4035-8496-258a16a73e91)

More importantly, the typical in-house application GUI guide (e.g. FLUX, NUI) also represents alpha as percentage.

![image](https://github.com/user-attachments/assets/66a83cba-d902-498d-b74e-494de4f914cb)
![image](https://github.com/user-attachments/assets/cb50fbd5-bea8-498c-818d-f2783257b800)



Previously the NUI application developers with those guide had to choose one of followings,
1. Converts hex rgb to percentage value : `new Color(RR/255, GG/255, BB/255, a)`
2. Convects alpha to hex value: `new Color("#RRGGBB" + ((uint)(255*a)).ToString("X2"))`
3. Creates color with rgb and then sets the alpha : `new Color("#RRGGBB") { A = a };`

I wanted to make it simple -> new L.Color(0xRRGGBB, a);
(FYI, the constructor with string parameter will support "#RGBA" format later like `NUI.Color`)

Reply:
Okay. If it is trend of these days, it looks okay to me.